clients/ops/admin-ui/src/features/datastore-connections/add-connection/sass/ConnectorParameters.tsx

```diff
@@ -0,0 +1,289 @@
+    }
+  };
+
+  useEffect(() => {
+    mounted.current = true;
+    if (result.isSuccess) {
+      onTestConnectionClick(result);
+    }
+    return () => {
+      mounted.current = false;
+    };
+  }, [onTestConnectionClick, result]);
+
+  return (
+    <Formik
+      initialValues={getInitialValues()}
+      onSubmit={handleSubmit}
+      validateOnBlur={false}
+    >
+      {(props) => (
+        <Form noValidate>
+          <VStack align="stretch" gap="24px">
+            {/* Name */}
+            <Field
+              id="name"
+              name="name"
+              validate={(value: string) => validateField("Name", value)}
+            >
+              {({ field, form }: { field: any; form: any }) => (
+                <FormControl
+                  display="flex"
+                  isRequired
+                  isInvalid={form.errors.name && form.touched.name}
+                >
+                  {getFormLabel("name", "Name")}
+                  <VStack align="flex-start" w="inherit">
+                    <Input
+                      {...field}
+                      autoFocus
+                      color="gray.700"
+                      placeholder={`Enter a friendly name for your new ${capitalize(
+                        connectionOption.identifier
+                      )} connection`}
+                      size="sm"
+                    />
+                    <FormErrorMessage>{form.errors.name}</FormErrorMessage>
+                  </VStack>
+                  <CircleHelpIcon marginLeft="8px" />
+                </FormControl>
+              )}
+            </Field>
+            {/* Description */}
+            <Field id="description" name="description">
+              {({ field }: { field: any }) => (
+                <FormControl display="flex">
+                  {getFormLabel("description", "Description")}
+                  <Textarea
+                    {...field}
+                    color="gray.700"
+                    placeholder={`Enter a description for your new ${capitalize(
+                      connectionOption.identifier
+                    )} connection`}
+                    resize="none"
+                    size="sm"
+                  />
+                  <CircleHelpIcon marginLeft="8px" />
+                </FormControl>
+              )}
+            </Field>
+            {/* Connection Identifier */}
+            <Field id="instance_key" name="instance_key">
+              {({ field }: { field: any }) => (
+                <FormControl display="flex" isRequired>
+                  {getFormLabel("instance_key", "Connection Identifier")}
+                  <Input
+                    {...field}
+                    color="gray.700"
+                    placeholder={`A a unique identifier for your new ${capitalize(
+                      connectionOption.identifier
+                    )} connection`}
+                    size="sm"
+                  />
+                  <Tooltip
```

Comment:
It looks like the tooltip SVG is being shown regardless of whether this element is present

![Screenshot 2022-08-18 at 15 08 41](https://user-images.githubusercontent.com/1667340/185474868-e852b2ba-d376-4103-aaa2-84a4dfe31fcc.png)

Reply:
I believe the question mark circle icon is to be a placeholder for additional helpful hints to be provided via the API at some point. This was illustrated in the original Figma designs. @adriaaaa Should these icons not be displayed until we have this API functionality implemented? Please correct me.

Reply:
hey @chriscalhoun1974 yeah on the ticket we said to ignore the tooltips on each row. We actually may not need them if we implement this [ticket](https://github.com/ethyca/fidesops/issues/887).

Reply:
I have removed all of the tooltip icons except for the Connection Identifier input.